src/ui/public/vislib/visualizations/point_series/column_chart.js

```diff
@@ -10,8 +10,11 @@ export function VislibVisualizationsColumnChartProvider(Private) {
     mode: 'normal',
     showTooltip: true,
     color: undefined,
-    fillColor: undefined
+    fillColor: undefined,
   };
+
+  const minBarWidth = 1;
```

Comment:
@ppisljar, "large enough" is when it looks good.  I've created panels that are the full width of the dashboard with 10+ items in it horizontally (I think the largest I did was about 20 line charts).  I've also created panels that are really tall with vertical splits.  Obviously a tiny panel makes no sense with lots of splits but limiting the number of splits just because of tiny panels is not right.

Is it even worth it to care about the "too small" message?  Can't Kibana just try and fit the graph in the space all the time?  As the dashboard creator, I'll notice if the panel doesn't "fit" because it won't look good (text is cut off, graph is too small, etc).  But I can easily fix that by resizing the panel.